File: python_programming_basics/conditional_statements _if_else_elif_debug/e_04_toy_shop.py
trip_price = float(input())
puzzels_count = int(input())
dolls_count = int(input())
bears_count = int(input())
minions_count = int(input())
trucks_count = int(input())

toys_price = puzzels_count * 2.6 + dolls_count * 3 + bears_count * 4.1 + minions_count * 8.2 + trucks_count * 2
toys_count = puzzels_count + dolls_count + bears_count + minions_count + trucks_count

if toys_count >= 50:
    discount = toys_price * 0.25
else:
    discount = 0

total_price = toys_price - discount
rent_price = total_price * 0.1
profit = total_price - rent_price

if profit >= trip_price:
    print(f"Yes! {profit - trip_price:.2f} lv left.")
else:
    print(f"Not enough money! {trip_price - profit:.2f} lv needed.")

File: JavaScript Basics/02. Conditional Statements/e-04-Toy-Shop.js
function solve(tripPrice, puzzles, dolls, teddys, minions, trucs) {
    let toysPrice = puzzles * 2.6 + dolls * 3 + teddys * 4.1 + minions * 8.2 + trucs * 2;
    let toysCount = puzzles + dolls + teddys + minions + trucs;

    if(toysCount >= 50) {
        toysPrice *= 0.75;
    }

    toysPrice *= 0.9;

    if (toysPrice >= tripPrice) {
        let moneyLeft = toysPrice - tripPrice;
        console.log(`Yes! ${moneyLeft.toFixed(2)} lv left.`);
        
    } else {
        moneyNeeded = tripPrice - toysPrice
        console.log(`Not enough money! ${moneyNeeded.toFixed(2)} lv needed.`);
        
    }
}

solve(40.8,
    20,
    25,
    30,
    50,
    10
    )